usaspending_api/accounts/v2/filters/object_class_program_activity.py

```diff
@@ -36,26 +35,26 @@ def object_class_program_activity_filter(filters):
     else:
         raise InvalidParameterException('fy and quarter are required parameters')
 
-    queryset = FinancialAccountsByProgramActivityObjectClass.objects
-
-    # TODO: Derivations
-    # ata_subquery = ToptierAgency.objects.filter(cgac_code=OuterRef('treasury_account__allocation_transfer_agency_id'))
-    # agency_name_subquery = ToptierAgency.objects.filter(cgac_code=OuterRef('treasury_account__agency_id'))
-    # queryset = queryset.annotate(
-    #     treasury_account_symbol=Concat(
-    #         'treasury_account__agency_id',
-    #         Value('-'),
-    #         Case(When(treasury_account__availability_type_code='X', then=Value('X')),
-    #              default=Concat('treasury_account__beginning_period_of_availability', Value('/'),
-    #                             'treasury_account__ending_period_of_availability'),
-    #              output_field=CharField()),
-    #         Value('-'),
-    #         'treasury_account__main_account_code',
-    #         Value('-'),
-    #         'treasury_account__sub_account_code',
-    #         output_field=CharField()),
-    #     allocation_transfer_agency_name=Value(Subquery(ata_subquery.values('name')[:1]), output_field=CharField()),
-    #     agency_name=Value(Subquery(agency_name_subquery.values('name')[:1]), output_field=CharField())
-    # )
+    # Derive treasury_account_symbol, allocation_transfer_agency_name, and agency_name
```

Comment:
Seems like similar code is being repeated several times. Is there any way to refactor these derivations? If it's too much of a hassle, it can be ignored.